PluginCode/src/main/java/smr/cs/ualberta/libcomp/data/Statement.java

```diff
@@ -0,0 +1,25 @@
+package smr.cs.ualberta.libcomp.data;
+
+/**
+ * The DataStatement class is a parent class for ImportStatement and DependencyStatement
```

Comment:
It's only Statement not DataStatement.. perhaps add a description of it.. like it is any statement that LibComp may want to replace?